AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -158,6 +158,7 @@ rtp.pt###rtpprivacycontent
 fansale.de###thirdPartyCookiesLayer
 speedtest.net###privacy-updated-modal
 criticker.com###i_cookies
+sosfanta.calciomercato.com##.modal-content
```

Comment:
I think that it would be better to add `sosfanta.calciomercato.com###gdpr-law`, otherwise is impossible to click any link until you scroll down a website.
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/55954821-03178680-5c60-11e9-8329-eb8665a7714b.png)

</details>